enterprise/embed-media/index.md

```diff
@@ -24,4 +24,7 @@ If you'd rather deploy Enhanced Media Embed via the Self-hosted package, you hav
 
 You also need to [integrate]({{ site.baseurl }}/enterprise/embed-media/mediaembed-server-integration/) and [configure]({{ site.baseurl }}/enterprise/embed-media/mediaembed-server-config/) the Enhanced Media Embed server.
 
-> **Important note:** The Enhanced Media Embed server currently does not support integration with IBM WebSphere Application Server.
+{% assign pluginname = 'Enhanced Media Embed' %}
```

Comment:
Looking at the rendered page, the content flow doesn't quite make sense? It jumps from cloud to self-hosted back to cloud.

Reply:
swapped the two sections around, with a _minor_ tidy-up on the self-hosted section.